docs/general/ad-filtering/create-own-filters.md

```diff
@@ -218,7 +219,9 @@ The following modifiers are the most simple and frequently used.
 
 #### **`$domain`** {#domain-modifier}
 
-`$domain` limits the rule application area to a list of domains and their subdomains. To add multiple domains to one rule, use the `|`  character as a separator.
+`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains 
+(as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header).
```

Comment:
should it be a separate line?

Reply:
Line breaks are not rendered in markdown, I put them to avoid extremely long lines that go off screen in the editor :)